chromium_src/base/feature_list.cc

```diff
@@ -108,12 +100,21 @@ bool FeatureList::IsFeatureOverridden(const std::string& feature_name) const {
   return false;
 }
 
+// static
+FeatureState FeatureList::GetDefaultOrOverriddenFeatureState(
+    const Feature& feature) {
+  const auto& default_state_overrides = internal::GetDefaultStateOverrides();
+  const auto default_state_override_it = default_state_overrides.find(feature);
+  return default_state_override_it != default_state_overrides.end()
+             ? default_state_override_it->second
+             : feature.default_state;
+}
+
 }  // namespace base
 
 // This replaces |default_state| compare blocks with a modified one that
 // includes the state override check.
-#define default_state \
-  name&& internal::GetDefaultOrOverriddenFeatureState(feature)
+#define default_state name&& GetDefaultOrOverriddenFeatureState(feature)
```

Comment:
Could we try to find a better name for that function? Like `GetDefaultOrCompileOverriddenFeatureState` or `GetCompileTimeFeatureState`?

In fact, this one is about compile-time overrides.
The next one `IsFeatureOverridden` also include run-time overrides.
So they have similar names but check the different things.

Reply:
Thanks, renamed to `GetCompileTimeFeatureState`.